src/cli/commands/abstract.command.ts

```diff
@@ -1,12 +1,42 @@
 import Printer from '../lib/printer/printer';
+/* eslint-disable @typescript-eslint/no-require-imports */
+import loading = require('loading-cli');
 
 export class AbstractCommand {
 	protected static endProcess(status: number) {
 		process.exit(status);
 	}
 
+	total: number;
+	step: number;
+
 	protected printSuccess = Printer.format({fontColor: 'green', decoration: 'bold'});
 	protected printError = Printer.format({fontColor: 'red', decoration: 'bold'});
 	protected printNeutral = Printer.format({decoration: 'bold'});
+
+	constructor() {
+		this.total = 7;
+		this.step = 1;
+	}
```

Comment:
I wouldn't add these variables as part of the AbstractCommand. It's something very specific and should be managed out of this scope.